infra/gp-cluster-monitoring-config/helper/testalert.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
wird der noch gebraucht? Sollte meiner Meinung nach eher gelöscht werden, oder zumindest nicht in den Critical sondern in den anderen Channel wandern.

Reply:
wird nicht eingespielt weil helper ignoriert. soll als template dienen um schnell einen alert anzulegen der immer feuert.